content/blog/move-resources-between-stacks/index.md

```diff
@@ -0,0 +1,93 @@
+# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
+# https://twitter.com/PulumiCorp/status/1763265391042654623
+# https://twitter.com/PulumiCorp/status/1762900472489185492
+# https://twitter.com/PulumiCorp/status/1755637618631405655
+
+social:
+    twitter:
+    linkedin:
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+Today we're announcing the [`pulumi state move`](https://www.pulumi.com/docs/cli/commands/pulumi_state_move/) command, which can be used to move resources that are managed by Pulumi between different stacks and/or projects.  No matter in which stack/project the resource was created in, it can now be moved with a single commmand.
+
+<!--more-->
+
+When refactoring a Pulumi project from a [monolithic](https://www.pulumi.com/docs/using-pulumi/organizing-projects-stacks/#monolithic) structure to [micro stacks](https://www.pulumi.com/docs/using-pulumi/organizing-projects-stacks/#micro-stacks), you might need to move resources between different projects or stacks, without recreating them. While this has been possible with some major surgery (our users came up with [automated solutions](https://github.com/pulumi/pulumi/issues/3389#issuecomment-679020482)), it's fairly error prone and difficult.  This makes refactoring Pulumi programs quite cumbersome, and having a simpler, integrated solution for this problem was a [much requested feature](https://github.com/pulumi/pulumi/issues/3389).
+
+The `pulumi state move` command works as follows:
+
+```
+$ pulumi state move --help
+Move resources from one stack to another
+
+This command can be used to move resources from one stack to another. This can be useful when
+splitting a stack into multiple stacks or when merging multiple stacks into one.
+
+Usage:
+  pulumi state move [flags] <urn>...
+
+Flags:
+      --dest string       The name of the stack to move resources to
+  -h, --help              help for move
+      --include-parents   Include all the parents of the moved resources as well
+      --source string     The name of the stack to move resources from
+  -y, --yes               Automatically approve and perform the move
+```
+
+Both `dest` and `source` can be either stacks in the current project, or stacks in a different project, using the fully qualified stack names.  Note that this works only for stacks within the same backend, it is currently not possible to move a resource between different backends.
+
+The resources being moved have to be specified by their full URN, and multiple URNs can be passed at once.  For each resource being moved, all the children of that resource will also be moved, and the relationships between all resources being moved is preserved.  Resources with other types of dependencies will however not be moved to the target stack by default.
```

Comment:
Not sure if you want to get into it, but it might be worth mentioning that you may need to surround the URN with single quotes, as some URNs contain `$`, which can be misinterpreted by the shell.

Reply:
Yes, I think this is worth mentioning. Done :heavy_check_mark: